lib/currency/index.js

```diff
@@ -32,41 +46,44 @@ var symbols = {
     format: '$0,0.00',
     raw: '0,0.00'
   }
-};
+}; // eslint-disable-next-line no-unused-vars
 
 function parseLocaleNumber(stringNumber) {
   var thousandSeparator = 1111 .toLocaleString().replace(/1/g, '');
```

Comment:
Is that a space between 1111 and toLocaleString()?

Reply:
Looks like it - I'm unsure why Babel needed to remove the parens like that ([repl example](https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=GYVwdgxgLglg9mABABwIYCcDOBTAMnCVAG2wDkQBbAI23QApMp0YwBzc62gSkQG8AoRIggJGiKAAs4ITKjAATAMrY06VFDjpEAXkR0AjIf1cAdBvyESipi1Z1T6FUVQRsdAPT73rADSIA5P5cgsKiUIjy2BAwFMTKquqaOnr6JsZmcBbE2NbMbPYmjsjOrh5evgFB_CGOUCDoSKo4AGJEcOp0IUKMeeyUNOhdQoiFTi5uYNgA7ogAStisAKIAHsh0AAYAOpsAJLyS0rIK8RiJ6AC-637-rEHXVcPDo8XjdJMz80urG9t7kdGxIgnNQaC7rLjXEwPRDBc5AA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=7.7.3&externalPlugins=)).

It looks like that function may have only been used to test something and can be removed at this point. Thoughts?